### PR TITLE
Bug: Update map links in Contact component to use specific locations

### LINKED ISSUE #41: Maps links don't point to the correct location
| Google | Apple |
| --- | --- |
| Pin is on TecSalud esplanade | Pin is on TecSalud |
| Image | Image |

Links should be the following:
- Google: https://maps.app.goo.gl/vaAGnbzTkCvCTm2eA 
- Apple: https://maps.apple.com/?ll=20.733693,-103.456489&q=Habitat%20del%20Futuro%20de%20los%20Negocios&t=m

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
## Description
fixes #41 

fixes maps link in repo

<what this pull request is doing>
<what will be done in later PRs and not included here>

## Milestones
<the milestones or stories/features that this works towards>

## Resources
<links to tutorials, code snippets, inspirations>
<if you took or translated specific code from the internet, please call it out here!>

## Test Plan
<insert images or gifs of feature>
<otherwise, say how code was tested if not UI facing>
<any edge cases you might have specifically tested>

### CODE REVIEW

**Comment by hanrodz** [comment]
Thank you for contributing!

File: frontend/src/components/Contact.tsx [hunk 1]
```diff
@@ -17,10 +17,10 @@ const Contact: React.FC = () => {
             <p>Tecnológico de Monterrey, Campus Guadalajara</p>
             <p>Av. Gral. Ramón Corona 2514<br />Colonia Nuevo México, 45201<br />Zapopan, Jal., México</p>
             <div className="maps-links">
-                <a href="https://maps.app.goo.gl/pZdnoWDcySrFbRao6" target="_blank" rel="noopener noreferrer">
+                <a href="https://www.google.com/maps/place/H%C3%A1bitat+del+Futuro+de+los+Negocios/@20.7336346,-103.456528,21z/data=!4m6!3m5!1s0x8428a922f44685c3:0x83f5ed7fc747dde8!8m2!3d20.7336933!4d-103.4564891!16s%2Fg%2F11l37857y1!5m1!1e2?entry=tts&g_ep=EgoyMDI1MDMyNS4xIPu8ASoASAFQAw%3D%3D&skid=62df9a2c-5ef2-4c4e-8c47-68bcad04b6be" target="_blank" rel="noopener noreferrer">
                     <Image src={img_googlemaps} alt="Google Maps" />
                 </a>
-                <a href="https://maps.apple.com/?ll=20.733860,-103.456890&q=Instituto%20Tecnologico%20y%20de%20Estudios%20Superiores%20de%20Monterrey&t=m" target="_blank" rel="noopener noreferrer">
+                <a href="https://maps.apple.com/?address=ll=20.733693,-103.456489&q=Habitat%20del%20Futuro%20de%20los%20Negocios&t=m" target="_blank" rel="noopener noreferrer">
```
Comment (hanrodz) [inline frontend/src/components/Contact.tsx:23]: I made a mistake in the issue description and this link shouldn't include `address=`

File: frontend/src/components/Contact.tsx [hunk 2]
```diff
@@ -17,10 +17,10 @@ const Contact: React.FC = () => {
             <p>Tecnológico de Monterrey, Campus Guadalajara</p>
             <p>Av. Gral. Ramón Corona 2514<br />Colonia Nuevo México, 45201<br />Zapopan, Jal., México</p>
             <div className="maps-links">
-                <a href="https://maps.app.goo.gl/pZdnoWDcySrFbRao6" target="_blank" rel="noopener noreferrer">
+                <a href="https://www.google.com/maps/place/H%C3%A1bitat+del+Futuro+de+los+Negocios/@20.7336346,-103.456528,21z/data=!4m6!3m5!1s0x8428a922f44685c3:0x83f5ed7fc747dde8!8m2!3d20.7336933!4d-103.4564891!16s%2Fg%2F11l37857y1!5m1!1e2?entry=tts&g_ep=EgoyMDI1MDMyNS4xIPu8ASoASAFQAw%3D%3D&skid=62df9a2c-5ef2-4c4e-8c47-68bcad04b6be" target="_blank" rel="noopener noreferrer">
```
Comment (hanrodz) [inline frontend/src/components/Contact.tsx:20]: We should remove everything after the `?` in the URL

**Review by hanrodz — CHANGES_REQUESTED** [review]

**Comment by Brijeshthummar02** [comment]
@hanrodz i have made the changes, good to go?

**Review by hanrodz — APPROVED** [review]
LGTM! :)

**Comment by hanrodz** [comment]
Thanks for your contribution! :D

**Comment by Brijeshthummar02** [comment]
@hanrodz drop dm and mention anything that can be fixed by me, will do it.
